samples/matter/closure/src/persistent_percent.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "persistent_percent.h"
+
+#include <cstdlib>
+#include <cstring>
+#include <persistent_storage/persistent_storage.h>
+#include <platform/CHIPDeviceLayer.h>
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(persistent_percent, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace chip::DeviceLayer;
+
+PersistentPercent::PersistentPercent(const char *key, uint16_t defaultValue) : mKey(key), mValue(defaultValue) {}
```

Comment:
If the constructor has no body, please move it to the class in the same format as it is here.